Libraries/Modal/Modal.js

```diff
@@ -91,6 +103,18 @@ class Modal extends React.Component {
   }
 }
 
+Modal.propTypes = {
```

Comment:
Why are these prop types added here? It's already defined in the class as a static member.